### ISSUE TITLE
Running office2john.py AttributeError: 'ElementTree' object has no attribute 'getiterator'

### ISSUE THREAD

**Issue by paulcardelli** [openwall/john#4881]
Looks like it maybe due to a depreciated Attribute in Python used in the script.

Command and output:
D:\john-1.9.0-jumbo-1-win64\run>python office2john.py "test file.docx" > officepassword.txt
Traceback (most recent call last):
  File "D:\john-1.9.0-jumbo-1-win64\run\office2john.py", line 3082, in <module>
    ret = process_file(sys.argv[i])
  File "D:\john-1.9.0-jumbo-1-win64\run\office2john.py", line 3005, in process_file
    return process_new_office(filename)
  File "D:\john-1.9.0-jumbo-1-win64\run\office2john.py", line 2813, in process_new_office
    xml_metadata_parser(data, filename)
  File "D:\john-1.9.0-jumbo-1-win64\run\office2john.py", line 2857, in xml_metadata_parser
    for node in tree.getiterator('{http://schemas.microsoft.com/office/2006/keyEncryptor/password}encryptedKey'):
AttributeError: 'ElementTree' object has no attribute 'getiterator'

### Potential Fix might be related to the following ###

hint: https://docs.python.org/2/library/xml.et...etiterator (for python 2.7) it says:

getiterator(tag=None) : Deprecated since version 2.7: Use method ElementTree.iter() instead.

### IMPORTANT VERSION INFORMATION ###

Attach details about your OS and about john, including:
- The output of `./john --list=build-info`.
Version: 1.9.0-jumbo-1
Build: cygwin 64-bit x86_64 AVX2 AC OMP
SIMD: AVX2, interleaving: MD4:3 MD5:3 SHA1:1 SHA256:1 SHA512:1
CPU tests: AVX2
CPU fallback binary: john-xop
OMP fallback binary: john-avx2-non-omp
$JOHN is
Format interface version: 14
Max. number of reported tunable costs: 4
Rec file version: REC4
Charset file version: CHR3
CHARSET_MIN: 1 (0x01)
CHARSET_MAX: 255 (0xff)
CHARSET_LENGTH: 24
SALT_HASH_SIZE: 1048576
SINGLE_IDX_MAX: 2147483648
SINGLE_BUF_MAX: 4294967295
Effective limit: Number of salts vs. SingleMaxBufferSize
Max. Markov mode level: 400
Max. Markov mode password length: 30
gcc version: 7.4.0
OpenCL headers version: 2.2
Crypto library: OpenSSL
OpenSSL library version: 01010102f
OpenSSL 1.1.1b  26 Feb 2019
GMP library version: 6.1.2
File locking: fcntl()
fseek(): fseek
ftell(): ftell
fopen(): fopen
memmem(): System's

Running this on Windows 10 Python version 3.9.7

The password protected word.docx was create with latest MS 365 version. If need I can post a clean version of the file. I will also attempt to recreate in Kali and on my Mac. 

[test file.docx](https://github.com/openwall/john/files/7572601/test.file.docx)

**Comment by paulcardelli** [comment]
It appears to work fine in Kali, even used the same office2john.py script and file. So likely there is an issue with the python3 for Windows. So the work around it for me is to not use windows.

**Comment by ghost** [comment]
This was fixed at eb7cbf6e72ba4ad04a164fe832de51b59e94b6f9. 

Please use the bleeding-jumbo version from this repository.

```
$ /snap/john-the-ripper/current/run/office2john.py test.file.docx > officepassword.txt

$ cat officepassword.txt 
test.file.docx:$office$*2013*100000*256*16*704122c720b7c2cc9d4bfaf28e0f5c6a*0b40809b8e77f3611b8ed109a8817a72*079b44fed88b3024a3832bba60bb719091283eb9236f37ae4718412be21d5640

```

**Comment by paulcardelli** [comment]
perfect thank you, for the quick response
